src/TulsiGenerator/PBXTargetGenerator.swift

```diff
@@ -1494,9 +1494,24 @@ final class PBXTargetGenerator: PBXTargetGeneratorProtocol {
     } else {
       changeDirectoryAction = "cd \"\(workingDirectory)\""
     }
-    let shellScript = "set -e\n" +
+
+    let shellScript: String
+    if let templatePath = options[.BuildActionScriptTemplate].commonValue {
+      // Load the user specified template. The tempalte path is relative to the project
```

Comment:
Extract this branch to a method and cover with a unit test.

Reply:
Ok thanks for suggestion, will do.